dance/datasets/spatial.py

```diff
@@ -175,8 +234,10 @@ def _load_raw_data(self) -> Tuple[pd.DataFrame, ...]:
                 continue
             elif ext == ".csv":
                 raw_data_dict[filename] = pd.read_csv(filepath, header=0, index_col=0)
+                raw_data_dict[filename].index = raw_data_dict[filename].index.astype(str)
```

Comment:
There seems to be several places where you manually enforced str type index. Could you comment on why this is necessary? (Are they not string type in the first place? Why do you have to use string type index)

Reply:
@xingzhongyu